database/postgres/utils.js

```diff
@@ -28,60 +28,31 @@ module.exports = {
         i += 1;
         let date = JSON.stringify(faker.date.past()).slice(1, 11)
         let review = {
-          title: faker.lorem.words().slice(0, 4),
+          title: faker.lorem.words().replace(/,/g, ""),
           abuseReported: faker.datatype.boolean(),
           rating: faker.datatype.number(5),
-          location_: faker.address.country(),
+          location_: faker.address.country().replace(/,/g, ""),
           userid: Math.floor((Math. random() * 1000000) + 1),
           productid: Math.floor((Math. random() * 1000000) + 1),
           reviewDate: date,
-          reviewBody: faker.lorem.paragraph().slice(0, 4),
+          reviewBody: faker.lorem.paragraph().replace(/,/g, ""),
           helpfulCount: faker.datatype.number(2000),
         }
         const data = `${review.title},${review.abuseReported},${review.rating},${review.location_},${review.userid},${review.productid},${review.reviewDate},${review.reviewBody},${review.helpfulCount}\n`;
         if (i === 40000000) {
-          writer.write(data.slice(-1), encoding, cb);
+          // if (i === 1000) {
+          writer.write(data, encoding, cb);
```

Comment:
Are you writing 40 million records to one file? Didn't realize we could do that many. Seems efficient